src/main.rs

```diff
@@ -256,10 +258,20 @@ fn parse_args(config: config::Config) {
         } => {
             if !config_dir_exists() {
                 warn_user!("Config directory '/etc/acs' does not exist!");
+                thread::sleep(time::Duration::from_millis(5000));
             }
 
+            let parsed_graph_type = match get_graph_type(&should_graph) {
+                Some(graph_type) => graph_type,
+                None => {
+                    warn_user!("Graph type is not set! Can be freq, usage, or temp Continuing in 5 seconds...");
```

Comment:
I think we should just let GraphType be hidden without telling the user that there is a warning

Reply:
Would it be possible to show the warning if the user does the -g flag without an arg to help the user out, but not if the don't include the flag?

Reply:
Ah yes, I see what you mean, the intended behavior was for the warning to show only if the graph arg is stated but the graph type is invalid or not set. Yes, I'll fix that thank you.

Reply:
Thanks!